pkg/daemon/pivot/types/imageinspection.go

```diff
@@ -3,4 +3,9 @@ package types
 const (
 	// PivotNamePrefix is literally the name prefix of the new pivot
 	PivotNamePrefix = "ostree-container-pivot-"
+
+	// PivotFailurePath is the path pivot writes its error to when it fails. We need this to bubble
+	// up any error happning there as we just log those errors in MCD and are useful unless we report
+	// them up the stack
```

Comment:
this isn't quite english, I know (rushed) I'll fix it while testing this whole thing out